crates/ruff_linter/src/rules/ruff/rules/unnecessary_fstring.rs

```diff
@@ -0,0 +1,92 @@
+use crate::checkers::ast::Checker;
+use ruff_diagnostics::{Diagnostic, Edit, Fix, FixAvailability, Violation};
+use ruff_macros::{derive_message_formats, ViolationMetadata};
+use ruff_python_ast::{self as ast, ConversionFlag};
+use ruff_text_size::Ranged;
+
+/// ## What it does
+/// Checks for f-strings that consist of a single format item.
+///
+/// ## Why is this bad?
+/// Wrapping the expression in extra quotes and curly braces makes the
+/// code harder to read.
+///
+/// ## Example
+/// ```python
+/// f"{variable}"
+/// ```
+///
+/// Use instead (if `variable` needs to be converted to a string explicitly):
+/// ```python
+/// str(variable)
+/// ```
+/// Or (if `variable` already has the desired string-like properties):
+/// ```python
+/// variable
+/// ```
+
+#[derive(ViolationMetadata)]
+pub(crate) struct UnnecessaryFString {
```

Comment:
We should probably use a more specific rule name to distinguish it from https://docs.astral.sh/ruff/rules/f-string-missing-placeholders/ (which you could argue, is also an unnecessary f-string). 

Maybe `single-placeholder-f-string`